entry.go

```diff
@@ -37,6 +37,7 @@ type EntryConfiguration struct {
 	ReverseSubscriptionPrefix   string                      `json:"ReverseSubscriptionPrefix"`
 	ReverseSubscriptionDuration uint32                      `json:"ReverseSubscriptionDuration"`
 	ReverseSubscriptionFee      string                      `json:"ReverseSubscriptionFee"`
+	IPFilter                    IPFilter                    `json:"IPFilter"`
```

Comment:
I think we need per-service IP filter. For example, as an entry, I may use 2 services: one for httpproxy (which I need an US IP address), and one for moonlight game streaming (which I need a China IP address for better latency). A single global IP filter is not enough.